locale/es.json

```diff
@@ -124,43 +124,43 @@
                 },
                 "delete": {
                     "title": "Borrar",
-                    "label": "Borra el vehiculo actual",
-                    "success": "¡Vehiculo borrado!"
+                    "label": "Borra el vehículo actual",
+                    "success": "¡Vehículo borrado!"
                 }
             },
             "heal": {
                 "title": "Curar",
                 "myself": {
-                    "title": "A mi mismo",
+                    "title": "A mí mismo",
                     "label": "Regenera tu vida",
                     "success_0": "¡Todo curado!",
-                    "success_1": "¡Deberias sentirte mejor!",
+                    "success_1": "¡Deberías sentirte mejor!",
                     "success_2": "¡A tope!",
                     "success_3": "¡Sana, sana, pata de rana!"
                 },
                 "everyone": {
                     "title": "A todos",
                     "label": "Cura y revive a todos",
-                    "success": "Todos los jugadores revividos y curados."
+                    "success": "Todos los jugadores han sido revividos y curados."
                 }
             },
             "announcement": {
                 "title": "Enviar anuncio",
-                "label": "Envia una comunicación a todos los jugadores conectados.",
-                "dialog_desc": "Enter the message you want to broadcast to all players.",
-                "dialog_placeholder": "Tu comunicación...",
+                "label": "Envía una comunicación a todos los jugadores conectados.",
```

Comment:
This could probably be changed to "comunicado" instead of comunicacion.

Reply:
You're completely right on that one! Will get that one updated to "un comunicado". Those small details!